docs/docs/forms.mdx

```diff
@@ -494,16 +494,3 @@ class AskForSlotAction(Action):
         return []
 ```
 
-## Debugging
```

Comment:
have we moved this stuff elsewhere or why are we removing this?

Reply:
It seems out of place here, and isn't really about debugging forms as much as testing the bot in general. I think it's a leftover from the previous docs structure.

Reply:
ok yeah let's just leave it out then